src/openrct2/ride/TrackPaint.cpp

```diff
@@ -2014,11 +2014,11 @@ void PaintTrack(PaintSession& session, Direction direction, int32_t height, cons
         {
             const int32_t rideClearance = rtd.Heights.ClearanceHeight;
             const int32_t trackClearance = ted.sequences[trackSequence].clearance.clearanceZ;
-            const int32_t supportClearance = std::max<int32_t>(
-                ted.sequences[trackSequence].clearance.supportClearanceZ,
+            const int32_t supportClearance = ted.sequences[trackSequence].clearance.supportClearanceZ;
```

Comment:
We should rename those in the future, how often do we have to mention clearance? I think something like `clearance.height` and `clearance.supportHeight` would work. Also not asking to have this changed in this PR, this is just a remark.